datasets/xsum/README.md

```diff
@@ -7,6 +7,33 @@ task_categories:
 - summarization
 task_ids:
 - news-articles-summarization
+train-eval-index:
+- config: default
+  task: text-generation
+  task_id: summarization
+  splits:
+    train_split: train
+    eval_split: test
+  col_mapping:
+    document: text
+    summary: target
+  metrics:
```

Comment:
Same comment as in previous PRs about using a single ROUGE entry for the metrics